docs/source/geometry.conversions.rst

```diff
@@ -50,3 +50,15 @@ Angle Axis
 
 .. autofunction:: angle_axis_to_quaternion
 .. autofunction:: angle_axis_to_rotation_matrix
+
+Pose (extrinsics)
+----------
+
+.. autofunction:: extrinsics_from_Rt
+.. autofunction:: Rt_from_extrinsics
+.. autofunction:: camtoworldRt_to_poseRt
+.. autofunction:: poseRt_to_camtoworldRt
+.. autofunction:: camtoworld_graphics_to_vision
+.. autofunction:: camtoworld_vision_to_graphics
```

Comment:
`from_cv_camera_pose_world` and make explicit in the docs that will convert to "graphics"

Reply:
maybe we could make explicit that the change is changing the direction of the `y` vector. Some kind of simetry without being a simetry XD

Reply:
Yes.
Oh, crap, I have realized, that there is a 3rd possible convention, besides camtoworld and worldtocam, the rotation can be from worldtocam, but the translation is negative center position in world coords. 
It would be good to add all of them and write about differences explicitly